digits/job.py

```diff
@@ -202,3 +211,44 @@ def disk_size_fmt(self):
         size = fs.get_tree_size(self._dir)
         return sizeof_fmt(size)
 
+    def get_tasks_recursively(self):
+        tasks = []
+        for task in self.tasks:
+            if task:
+                tasks.append(task)
+                tasks += task.get_tasks_recursively()
+        return tasks
```

Comment:
I'm confused by this function. If you have tasks A and B, and A is B's parent, wouldn't this function return `[A,B,A]`? Is that what you want? Don't you just want the list at `job.tasks`?